deps.edn

```diff
@@ -475,7 +475,9 @@
                ["metabase\\.driver\\.mysql.*"
                 "metabase\\.driver\\.postgres.*"]}
    ;; different port from `:test` so you can run it at the same time as `:test`.
-   :jvm-opts ["-Dmb.jetty.port=3002"]}
+   :jvm-opts ["-Dmb.jetty.port=3002"
+              ;; default heap size is 1GB
```

Comment:
Maybe add WHY we're doing this. @qnkhuat

Reply:
ack, let's see if it works first

Reply:
if we're using an 8GB machine, we should give the JVM more than a 2GB heap size... we're leaving a lot of RAM on the Table